mmdet/structures/bbox/base_boxes.py

```diff
@@ -509,6 +509,17 @@ def find_inside_points(self,
         """
         pass
 
+    @abstractmethod
+    def create_masks(self, img_shape: Tuple[int, int]) -> Tensor:
```

Comment:
Is the abstract method reasonable? It would force all downstream classes to be implemented. I don't think it should be an abstract method

Reply:
I intended to alert users when the new [`bbox_type`](https://github.com/open-mmlab/mmrotate/pull/877) does not define `create_masks` when using `CopyPaste`.
And maybe it would be useful to support some tasks such as box-supervised instance segmentation.

But, I agree with the trade-off.
Is it desirable to remove `create_masks` in `BaseBoxes` or just `@abstractmethod` decorator?

Reply:
i think remove `create_masks` in BaseBoxes are better